processing-tests/common/test/kotlin/com/livefront/sealedenum/compilation/visibility/PrivateInterfaceSealedClass.kt

```diff
@@ -0,0 +1,132 @@
+package com.livefront.sealedenum.compilation.visibility
+
+import com.livefront.sealedenum.GenSealedEnum
+import org.intellij.lang.annotations.Language
+
+sealed class PrivateInterfaceSealedClass :
+    JavaPrivateInterfaceSubclass(JavaPrivateInterfaceOuterClass()), PrivateInterface {
+
+    object FirstObject : PrivateInterfaceSealedClass()
+
+    object SecondObject : PrivateInterfaceSealedClass()
+
+    @GenSealedEnum(generateEnum = true)
+    companion object
+}
+
+private interface PrivateInterface
+
+@Language("kotlin")
+val privateInterfaceSealedClassGenerated = """
+package com.livefront.sealedenum.compilation.visibility
+
+import com.livefront.sealedenum.EnumForSealedEnumProvider
+import com.livefront.sealedenum.SealedEnum
+import com.livefront.sealedenum.SealedEnumWithEnumProvider
+import java.lang.Class
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.List
+
+/**
+ * An isomorphic enum for the sealed class [PrivateInterfaceSealedClass]
+ */
+public enum class PrivateInterfaceSealedClassEnum {
+    PrivateInterfaceSealedClass_FirstObject,
+    PrivateInterfaceSealedClass_SecondObject,
+}
+
+/**
+ * The isomorphic [PrivateInterfaceSealedClassEnum] for [this].
+ */
+public val PrivateInterfaceSealedClass.`enum`: PrivateInterfaceSealedClassEnum
+    get() = PrivateInterfaceSealedClassSealedEnum.sealedObjectToEnum(this)
+
+/**
+ * The isomorphic [PrivateInterfaceSealedClass] for [this].
+ */
+public val PrivateInterfaceSealedClassEnum.sealedObject: PrivateInterfaceSealedClass
+    get() = PrivateInterfaceSealedClassSealedEnum.enumToSealedObject(this)
+
+/**
+ * An implementation of [SealedEnum] for the sealed class [PrivateInterfaceSealedClass]
+ */
+public object PrivateInterfaceSealedClassSealedEnum : SealedEnum<PrivateInterfaceSealedClass>,
+        SealedEnumWithEnumProvider<PrivateInterfaceSealedClass, PrivateInterfaceSealedClassEnum>,
+        EnumForSealedEnumProvider<PrivateInterfaceSealedClass, PrivateInterfaceSealedClassEnum> {
+    public override val values: List<PrivateInterfaceSealedClass> = listOf(
+        PrivateInterfaceSealedClass.FirstObject,
+        PrivateInterfaceSealedClass.SecondObject
+    )
+
```

Comment:
Minor, but I'd remove one of these extra lines.

Reply:
Per this comment and the rest, in these tests I'm checking the direct output of the generated files from `kotlinpoet`, so these formatting oddities are due to the code generation. If I did remove one of the lines here, then the test would fail due to generating mismatched code.